apisix/cli/snippet.lua

```diff
@@ -70,6 +70,15 @@ function _M.generate_conf_server(env, conf)
         end
     end
 
+    local ssl_trusted_certificate
+    local etcd_tls_verify = etcd.tls.verify
+    if enable_https and etcd_tls_verify then
+        if not conf.apisix.ssl.ssl_trusted_certificate then
+            return nil, "should set ssl_trusted_certificate if etcd tls verify is enabled"
```

Comment:
Now we can remove:
https://github.com/apache/apisix/blob/c397ede02d0b304b8b3a1d5049f6c1d8a83d3804/apisix/cli/snippet.lua#L42
?

Reply:
removed